app/prisma/migrations/20240114012208_reverse_the_order_of_relabel_request_unique_key/migration.sql

```diff
@@ -0,0 +1,11 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[datasetEntryPersistentId,batchId]` on the table `RelabelRequest` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- DropIndex
+DROP INDEX "RelabelRequest_batchId_datasetEntryPersistentId_key";
+
+-- CreateIndex
+CREATE UNIQUE INDEX "RelabelRequest_datasetEntryPersistentId_batchId_key" ON "RelabelRequest"("datasetEntryPersistentId", "batchId");
```

Comment:
Reversing this index to start with datasetEntryPersistentId speeds up the initial dataset entries loading time.